forte/data/ontology/top.py

```diff
@@ -891,6 +891,7 @@ def max_y(self):
 
     @property
     def image_shape(self):
+        self.pack.get_payload_data_at(Modality.Image, self._image_payload_idx)
```

Comment:
Let's note down the purpose of this line. Are we trying to validate some conditions or to load the image size?

Reply:
https://github.com/asyml/forte/pull/893

update the function in this PR